### DISCUSSION TITLE
Clicking on TOC entry on marp HTML output

### DISCUSSION THREAD

**Discussion by puat133** [marp-team/marp#278] [Q&A] [answered]
Hi,

If I click one of the toc entries in the marp HTML output, it does not take me to the relevant slide. Is this the expected behavior, or am I missing something?

Thanks

**Comment by yhatt** [comment, marked as answer]
~~Currently named anchor links are not working in HTML output, but~~ working in PDF output. Or anchor links to slide pages (`#1`, `#2`) will work. Try to use links to the slide page as ToC.

Related: #161

**Reply by yhatt** [reply]
**UPDATE**: The named anchor links `[link](#heading)` should be working now, by the latest update of Marp ecosystem. https://github.com/orgs/marp-team/discussions/161#discussioncomment-5627844
